server/MetaSnapper.cc

```diff
@@ -198,6 +200,14 @@ MetaSnappers::init()
 }
 
 
+void
+MetaSnappers::unload()
+{
+    for (iterator it = entries.begin(); it != entries.end(); ++it)
```

Comment:
NP: I'd prefer the new syntax without iterators and increment:
```c++
for(auto& entry : entries)
    entry.unload();
```

Reply:
Snapper still uses the old syntax since it was created mainly before C++11.

Reply:
OK, but could we use the new syntax while we are touching the code anyway? In YaST we also still have a lot of YCP-like Ruby code but we are gradually rewriting it to pure Ruby when touching it. Would this approach work also for snapper?

Reply:
OK, I have changed it to the new C++11 syntax. But given the current activity of the snapper project there will be a mixture of syntaxes for many years.